src/app/not-found/not-found.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+declare var particlesJS: any;
+
+@Component({
+  selector: 'mifosx-not-found',
+  templateUrl: './not-found.component.html',
+  styleUrls: ['./not-found.component.scss']
+})
+export class NotFoundComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+    particlesJS.load('particles-js', '/assets/particles.json', null);
```

Comment:
Do not pass a null argument, you can simply right it as `particlesJS.load('particles-js', '/assets/particles.json');` without any argument. Its not necessary and that should work.